src/test/groovy/com/intershop/gradle/icm/docker/ICMDockerTaskIntegrationSpec.groovy

```diff
@@ -25,12 +25,20 @@ class ICMDockerTaskIntegrationSpec extends AbstractIntegrationGroovySpec {
 
     def 'test create volumes task'() {
         settingsFile << """
+        pluginManagement {
+            repositories {
+        		gradlePluginPortal()
+                mavenCentral()
+                mavenLocal()
+            }
+        }
         rootProject.name='rootproject'
         """.stripIndent()
 
         buildFile << """
             plugins {
                 id 'java'
+                id 'com.intershop.gradle.icm.project' version '5.6.0-SNAPSHOT'
```

Comment:
SNAPSHOT?

Reply:
Fixed. Thanks. This is a bad solution for the test, but currently necessary.